apps/passport-client/components/shared/AppHeader.tsx

```diff
@@ -9,10 +9,15 @@ export const AppHeader = React.memo(AppHeaderImpl);
 
 function AppHeaderImpl({
   children,
-  isProveOrAddScreen = false
+  isProveOrAddScreen = false,
+  start
 }: {
   children?: React.ReactNode;
   isProveOrAddScreen?: boolean;
+  /**
+   * A React node to render on the left side of the header. If not provided, a default Info button will be rendered.
+   */
```

Comment:
worth clarifying whether in the case that the icon is there clicking on it causes the info modal to appear